.gitignore

```diff
@@ -1,5 +1,6 @@
 .idea
 *.iml
+docker-volumes
```

Comment:
Because I wanted it not to be gone after an sbt clean.

We talked about it and I leave this the default but allow configuration.